idea/jvm-debugger/jvm-debugger-test/test/org/jetbrains/kotlin/idea/debugger/test/IrKotlinSteppingTestGenerated.java

```diff
@@ -0,0 +1,1310 @@
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/manyFilesWithInlineCalls1.kt");
+        }
+
+        @TestMetadata("manyFilesWithInlineCalls2.kt")
+        public void testManyFilesWithInlineCalls2() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/manyFilesWithInlineCalls2.kt");
+        }
+
+        @TestMetadata("severalFunLiterals.kt")
+        public void testSeveralFunLiterals() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/severalFunLiterals.kt");
+        }
+
+        @TestMetadata("severalFunLiteralsInClass.kt")
+        public void testSeveralFunLiteralsInClass() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/severalFunLiteralsInClass.kt");
+        }
+
+        @TestMetadata("severalInlineCallsFromOtherFile.kt")
+        public void testSeveralInlineCallsFromOtherFile() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/severalInlineCallsFromOtherFile.kt");
+        }
+
+        @TestMetadata("severalInlineFunctionsInOneFile.kt")
+        public void testSeveralInlineFunctionsInOneFile() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/severalInlineFunctionsInOneFile.kt");
+        }
+
+        @TestMetadata("simpleConditionalBreakpoint.kt")
+        public void testSimpleConditionalBreakpoint() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/simpleConditionalBreakpoint.kt");
+        }
+
+        @TestMetadata("smartStepIntoComponentFunction.kt")
+        public void testSmartStepIntoComponentFunction() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/smartStepIntoComponentFunction.kt");
+        }
+
+        @TestMetadata("smartStepIntoConstructor.kt")
+        public void testSmartStepIntoConstructor() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/smartStepIntoConstructor.kt");
+        }
+
+        @TestMetadata("smartStepIntoFunWithDefaultArgs.kt")
+        public void testSmartStepIntoFunWithDefaultArgs() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/smartStepIntoFunWithDefaultArgs.kt");
+        }
+
+        @TestMetadata("smartStepIntoInlinedFunLiteral.kt")
+        public void testSmartStepIntoInlinedFunLiteral() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/smartStepIntoInlinedFunLiteral.kt");
+        }
+
+        @TestMetadata("smartStepIntoInlinedFunctionalExpression.kt")
+        public void testSmartStepIntoInlinedFunctionalExpression() throws Exception {
+            runTest("idea/jvm-debugger/jvm-debugger-test/testDat" +
```

Comment:
Please regenerate file

Reply:
Seem that changes in Fir** tests already  pushed to master

Reply:
Whoops, yes, rebased.